src/libs/antares/study/parts/short-term-storage/cluster.cpp

```diff
@@ -75,4 +76,9 @@ bool STStorageCluster::loadSeries(const std::string& folder) const
     return ret;
 }
 
+bool STStorageCluster::validateInitialLevel()
```

Comment:
About `STStorageCluster::validateInitialLevel()` member function :
As you know, the initial level is a cluster property.
A lot of cluster properties are checked in **STStorageCluster::validateInitialLevel()**, and so is the initial level, though it remains some checks to do on the  initial level.
So I'm wondering about this **STStorageCluster::validateInitialLevel()** function : what will it do exactly ?